### DISCUSSION TITLE
用AI臉部辨識追蹤新冠肺炎確診者

### DISCUSSION THREAD

**Discussion by chun1203** [1101-GS4719-C/Python-Community-GS4719-C#198] [科技新知]
[韓國近期將試行人工智慧、臉部辨識和上萬台閉路電視攝影機追蹤COVID-19確診者足跡計畫，然而此舉引發侵害隱私的疑慮。](https://www.bnext.com.tw/article/66755/face-track-covid-kr)
我認為利用AI追蹤確診者的確是相當有效率的方式，也可**分析是否有戴口罩、與確診者的距離**等等。
但利用人工智慧辨識難免會讓人有**隱私權**的疑慮，讓人覺得一舉一動都被**監控**的感覺。
如何在科技與隱私之間找出解決的方案或取得平衡的確是值得讓人深思的課題。

**Comment by Jie2222** [comment]
像是台灣，個資法的出現遠早於AI問世，規範不完備，沒有專責的機構都是很大的問題

**Reply by HSU-YU-JEN** [reply]
有道理~政府的政策有時真的需要更快的去反應科技新發展

**Comment by enoch-lightning** [comment]
我覺得AI發展到最後，可能會讓各個國家都能監控人民的一舉一動，人民會無所遁形。

**Comment by ChinChun0220** [comment]
在日常生活中非常容易被收集自己的臉部資料，像最近的體溫站需要偵測到臉部後才會量體溫，對於個資比較敏感的就會感覺不舒服，不知道會不會被收集data

**Comment by ghost** [comment]
之前有看過的影集也有應用到影像辨識來預測犯罪，未來隨著科技的持續進步，個資法等等的法規應該也會慢慢逐漸修正?
影集名稱:[疑犯追蹤](https://zh.wikipedia.org/wiki/%E7%96%91%E7%8A%AF%E8%BF%BD%E8%B8%AA)

**Comment by Tomlord1122** [comment]
也有一篇新聞報導了中國的天眼計畫，會來追蹤中國人他們的行為來記點，有好行為會加分，壞行為會扣分，雖然不知道被扣分會怎樣，但還是怕怕的

**Comment by EricAgumonJou** [comment]
我蠻好奇整形過後的人，會不會無法追蹤到ＸＤ

**Reply by enoch-lightning** [reply]
這是個既有趣，又實際的好問題。

**Comment by syt-110** [comment]
感覺有隱私權問題

**Comment by 109802503gloria** [comment]
人工智慧跨域學分學程有開蠻多課在討論人工智慧的議題，例如:人工智慧倫理導論、AI與法律、AI的社會運用與影響等，大家有興趣都能修~

**Reply by JAIMEHUNG** [reply]
AI與社會創新也有針對議題探討，可以跟同學們進行深度的想法思辨！

**Comment by luo910227** [comment]
新的科技，可能帶來進步，也可能造成反效果，而制衡新科技的方式就是有完善的法規跟制度。如果科技跑在制度之前，就如同deepfake剛開始發展的時候，將會帶來許多問題，所以先做好完善的立法與試驗，才是根本之道。

**Comment by zixunn** [comment]
感覺現今科技的進步遠超法規的訂定，對於隱私權我同意台灣專門設置一個第三方機關來處理，第三方機構不受政府或商人利益影響。

**Comment by syt-110** [comment]
感覺影響到確診者的隱私

**Comment by cody910120** [comment]
把那些確定確診還出來趴趴造的一個一個抓出來扁

**Comment by mesakiyuta** [comment]
這樣被監視感覺隱私權都沒了，且很容易被拿去惡用，感覺在運作上需要相當多的規範。

**Comment by YuHengChen54** [comment]
臉部辨識、追蹤這些，感覺都跟隱私離不開關係，這真的很兩難

**Comment by pekoranoinu** [comment]
臉部辨識和追蹤已經違反人權了吧!

**Comment by chupochun** [comment]
如果真的使用在生活中 我會有一種隱私被侵犯的感覺
